source/_integrations/touchline.markdown

```diff
@@ -11,9 +11,15 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The `touchline` climate platform let you control [ROTH Touchline](https://www.roth-uk.com/en/roth-touchline.htm) floor heating thermostats from Roth.
+The `touchline` climate platform let you control [Roth EnergyLogic Touchline](https://www.roth-uk.com/en/roth-touchline.htm) floor heating thermostats from Roth.
 
-To set it up, add the following information to your `configuration.yaml` file:
+<div class='note'>
+This integration does not work with the newer Roth Softline and Touchline SL/PL.
+</div>
+
+{% include integrations/config_flow.md %}
+
+To manually set it up, add the following information to your `configuration.yaml` file:
```

Comment:
Support for manual configuration should be deprecated in the parent PR (and thus removed from documentation).